### ISSUE TITLE
Code Performance Improvement!

### ISSUE THREAD

**Issue by juan-park** [f-rumblefish/Multi-Label-Fashion-MNIST#1]
Thank you for your code! Just wanted to contribute back.
I made a tweak to the data generation code that improves speed by 100X.
Basically, instead of plotting and saving the plot, I directly appended the numpy arrays and saved it using opencv!
```
from keras.datasets import fashion_mnist
import matplotlib.pyplot as plt
import pandas as pd
import numpy as np
from tqdm import tqdm
import cv2
(X_train, y_train), (X_test, y_test) = fashion_mnist.load_data()

fashion_name = ['T-shirt/top',
                'Trouser',
                'Pullover',
                'Dress',
                'Coat',
                'Sandal',
                'Shirt',
                'Sneaker',
                'Bag',
                'Ankle boot']

labels = pd.DataFrame(columns=['index', 'fname', 'label', 'real_int', 'real_str'])

for i in tqdm(range(0, 10000)):
    labels_in_picture = [0] * 10
    real_int = ''
    real_str = ''
    idx = np.random.randint(0, 1000)
    img = X_train[idx]
    labels_in_picture[y_train[idx]] = 1
    real_int += str(y_train[idx])
    real_int += ', '
    real_str += str(fashion_name[y_train[idx]])
    real_str += ', '
    for j in range(2, 5):
        idx = np.random.randint(0, 1000)
        img = np.append(img, X_train[idx], axis=1)
        labels_in_picture[y_train[idx]] = 1

        real_int += str(y_train[idx])
        if j != 4:
            real_int += ', '

        real_str += str(fashion_name[y_train[idx]])
        if j != 4:
            real_str += ', '

    # print(labels_in_picture)
    fname = "data/"+"%d" % i + ".png"
    cv2.imwrite(fname, img)

    labels = labels.append({"index": i,
                            "fname": fname,
                            "label": labels_in_picture,
                            "real_int": real_int,
                            "real_str": real_str
                            }, ignore_index=True)

labels.to_csv("labels.csv", index=False)
```

**Comment by f-rumblefish** [comment]
Got it. Thanks for your code!
